types/accounting/accounting-tests.ts

```diff
@@ -3,6 +3,9 @@
 // Default usage:
 accounting.formatMoney(12345678); // $12,345,678.00
 
+// Stringified usage:
+accounting.formatMoney('$4394958309392.9401'); // $4,394,958,309,392.94
```

Comment:
Could you please show me such example in their documentation/code?
I was not able to find an example in their documentation: http://openexchangerates.github.io/accounting.js/